osx/KA-Lite/KA-Lite/AppDelegate.m

```diff
@@ -207,9 +220,14 @@ - (void) runTask:(NSString *)command {
 
 
 NSString *getDatabasePath() {
-    // Defaults to ~/.kalite/ folder so check there for now.
-    // TODO(cpauya): Use the KALITE_HOME env var if it exists.
-    NSString *database = @"~/.kalite/database/data.sqlite";
+    NSString *database;
+    NSString* envKaliteHomeStr = getEnvVar(@"KALITE_HOME");
+    database = [NSString stringWithFormat:@"%@%@", envKaliteHomeStr, @"/database/data.sqlite"];
```

Comment:
This line must be moved inside the `if` statement below because it's overwritten if `KALITE_HOME` env var is not set.